### ISSUE TITLE
`-p a=b` is fail on Windows 

### ISSUE THREAD

**Issue by mikoto2000** [treasure-data/digdag#677]
I try digdag on windows.
`-p a=b` is not work.

Environment
===========

- OS: Windows 10 Pro 64bit
- Java: 1.8.0.151
- digdag: 0.9.20

Work log
========

```powershell
PS> # create test directory
PS> cd ~/tmp
PS> mkdir digdagtest
PS> cd digdagtest

PS> # get digdag
PS> Invoke-WebRequest http://dl.digdag.io/digdag-latest.jar -OutFile digdag.bat

PS> # run sample with session
PS> .\digdag.bat init mydag
PS> cd mydag
PS> ..\digdag.bat run .\mydag.dig -p aaa=bbb
2017-11-01 04:36:52 +0900: Digdag v0.9.20
error: Parameter -p expected a value of the form a=b but got:aaa
```

Cause
=====

When `%*` in for statement, delimiter `=` is enable.

https://github.com/treasure-data/digdag/blob/8b8aa86bed9947e389a2c82fef7673f4efedc233/digdag-cli/src/main/sh/selfrun.sh#L17

Example:

```cmd
>cat test.bat
@echo off

for %%a in ( %* ) do (
    echo %%a
)

>test.bat aaa bbb -p aaa=bbb
aaa
bbb
-p
aaa
bbb
```

**Comment by mikoto2000** [comment]
`-p a=b` is impossible, because selfrun script is `goto label` can't work.

Workarround(Required fix #676)
==

no whitespace:

- cmd.exe : ``-p "a=b"``
- powershell : ``-p `"a=b`"``

with whitespace:

- cmd.exe : ``-p "a=b c"``
- powershell : ``-p `"a=b c`"`` or `-p a="b c"`

**Comment by mikoto2000** [comment]
I close this issue, because we can avoid this problem in the next workaround.

- Workaround of previous comment
- Using the `java -jar` command
